modules/ServerAPI/src/server/routes/AuthorRoutes.coffee

```diff
@@ -6,6 +6,7 @@ exports.setupAPIRoutes = (app, loginRoutes) ->
 	app.delete '/api/authors/:id', exports.deleteAuthor
 	app.post '/api/author', exports.saveAuthor
 	app.put '/api/author/:id', exports.updateAuthor
+	app.post '/api/updateAuthorRoles', exports.updateAuthorRoles
```

Comment:
I've exposed the same logic behind `/api/projects/updateProjectRoles` at `/api/updateAuthorRoles` because I find it more idiomatic / logical.